libvirt/tests/src/migration/sriov_migrate.py

```diff
@@ -484,9 +485,14 @@ def add_hostdev_device(vm_name, pf_pci, params):
                 cmd_result = vm_after_mig.cmd_output(cmd)
 
                 if re.findall('Destination Host Unreachable', cmd_result, re.M):
-                    test.fail("The network does not work well during "
-                              "the migration period. ping output: %s"
-                              % cmd_result)
+                    if conn_dur_migration:
+                        test.fail("The network does not work well during "
+                                  "the migration period. ping output: %s"
+                                  % cmd_result)
+                    else:
+                        logging.warning("The network does not work well during "
+                                        "the migration period. ping output: %s"
+                                        % cmd_result)
```

Comment:
For logging, we use '*args'. And for reducing code overlaps, how about changing like: 
```suggestion
                    err_msg = ("The network does not work well during "
                                      "the migration period. ping output: %s"
                                      % cmd_result)
                    if conn_dur_migration:
                        test.fail(err_msg)
                    else:
                        logging.warning(err_msg)
```